index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-US", xmlns="http://www.w3.org/1999/xhtml">
   <head>
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
-<title>Adobe PDF Library 18 Documentation</title>
+<title>Datalogics Developer Documentation</title>
```

Comment:
Consider the title "Datalogics Product Documentation" as opposed to Developer Documentation, so that the title what's on the page rather than who it is for.